nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -18,7 +18,7 @@ in
       rm $out/disk.raw
       popd
     '';
-    configFile = <nixpkgs/nixos/modules/virtualisation/google-compute-config.nix>;
+    configFile = ./google-compute-config.nix;
```

Comment:
This one is fine and sufficient - we want the file from this directory, not from a random `<nixpkgs>` channel.